alacritty/src/window.rs

```diff
@@ -310,16 +312,15 @@ impl Window {
     }
 
     #[cfg(not(any(target_os = "macos", target_os = "windows")))]
-    pub fn get_window_id(&self) -> Option<usize> {
+    pub fn x11_window_id(&self) -> Option<usize> {
         match self.window().xlib_window() {
             Some(xlib_window) => Some(xlib_window as usize),
             None => None,
         }
     }
```

Comment:
This is not necessary for this PR, but this match statement looks really unnecessary.

This should work:

```rust
self.window.xlib_window().map(|win| win as usize)
```

I think that should do the job already, would you mind changing that?